drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -441,6 +446,23 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
+    var forwardKeyboardHelpButtonTemplate = function(ctx) {
+        return h(
+            'button.unbutton.btn-default.btn-link.'+"keyboard-help-button",
+            {attributes: {tabindex: 0}},
```

Comment:
I think you should remove explicitly setting the tabindex on buttons--as you mentioned, it is not necessary. Furthermore, changing the type of element from button to span "for styling purposes" would introduce a host of other issues; the reason to use native html components is that you get the correct behavior for screen-readers and keyboard navigation, without having to use aria attributes to make something act "like a button".